src/_sass/components/_header.scss

```diff
@@ -15,12 +15,12 @@
 
 .header__content {
   min-height: $header-small-height;
-  padding: 0 2.5rem 0 1.25rem;
+  padding: 0 4rem 0 1.25rem;
 
   @include flex-container();
 
   .wrapper & {
-    padding: 1rem 0;
+    padding: 1rem 2rem 1rem 0;
```

Comment:
What's the logic here?  The screen-width at which this makes a difference seems to be between 1035px and 1099px -- is this significant for some reason?

Reply:
Happy to merge as-is, but unclear on the rationale for this change.

Reply:
Hmm well the goal was to avoid the `ELEMENTS` drop-down being clipped by the edge of the window at screen width >= 1024px, which seems to be working for these (admittedly cursory) tests.
![Screenshot 2020-03-05 11 05 40](https://user-images.githubusercontent.com/7329112/76017345-4c557200-5ed3-11ea-8d71-654a6f51434c.png)
![Screenshot 2020-03-05 11 15 50](https://user-images.githubusercontent.com/7329112/76017510-950d2b00-5ed3-11ea-9ecf-2bc5bfc6bc0f.png)